### PR TITLE
WDB-84/confirm email page

### PR DESCRIPTION
**Pull request by tmaciulis22**
image

### CODE REVIEW

File: frontend/src/components/ConfirmEmailCard/ConfirmEmailCard.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+export default function ConfirmEmailCard() {
+  return (
+    <Paper elevation={3}>
+      <CardContent
+        direction="column"
+        align="center"
+        justify="center"
+      >
+        <img
+          src={confirmEmailIll}
```
Comment (AArlauskas) [inline frontend/src/components/ConfirmEmailCard/ConfirmEmailCard.js]: I get how you named it, but it looks confusing

**Review by AArlauskas — APPROVED** [review]
I would consider one renaming, but besides that, looks good

File: frontend/src/App.js [hunk 2]
```diff
@@ -30,13 +30,11 @@ function App() {
...
-            <Route exact path="/confirm-email">
-              <ConfirmEmailPage />
-            </Route>
           </Route>
-          <Route exact path="/register">
-            <RegisterPage />
+          <Route exact path="/confirm-email">
+              <ConfirmEmailPage />
           </Route>
+          <Route exact path="/register" component={RegisterPage} />
```
Comment (kibartas) [inline frontend/src/App.js:37]: Odd one out?

**Review by kibartas — CHANGES_REQUESTED** [review]
Are you gonna implement destructuring? I thought this PR was supposed to include it. Other than that, LGTM

**Pull request merged**